lib/fog/core/errors.rb

```diff
@@ -11,8 +11,6 @@ def self.slurp(error, message = nil)
       end
     end
 
-    class MockNotImplemented < Fog::Errors::Error; end
```

Comment:
But it is still an exception. I believe this extraction is unnecessary and make it harder to maintain the codebase. Can we leave this code here?